runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/AbstractDokkaTask.kt

```diff
@@ -14,7 +14,7 @@ import org.jetbrains.dokka.plugability.Configurable
 
 
 abstract class AbstractDokkaTask : DefaultTask(), Configurable {
-    @Input
+    @Internal
```

Comment:
This change does not seem obvious to me. However, I also can't remember why this was marked as Input anyways? 🤔